ecs-cli/integ/cmd/compose.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2015-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/aws/amazon-ecs-cli/ecs-cli/integ"
+
+	"github.com/aws/amazon-ecs-cli/ecs-cli/integ/stdout"
+)
+
+// A Project is the configuration needed to create an ECS Service.
+type Project struct {
+	Name       string
+	ConfigName string
+}
+
+// NewProject creates a new Project for an ECS Service.
+func NewProject(name string, configName string) *Project {
+	return &Project{
+		Name:       name,
+		ConfigName: configName,
+	}
+}
+
+// TestServiceUp runs `ecs-cli compose service up` for a project.
+func TestServiceUp(t *testing.T, p *Project) {
+	// Given
+	args := []string{
+		"compose",
+		"--project-name",
+		p.Name,
+		"service",
+		"up",
+		"--create-log-groups",
```

Comment:
Can you confirm the cmd behavior when the log groups already exist? My understanding is they'll be created the first time this is run, but creation is silently (or with a warning) skipped once they exist. 

It doesn't look like we're asserting on the existence or name of the log groups later in the test, so I'm wondering how useful this is to include (here and in the docker-compose.yml)?

Reply:
You're right it skips it with a WARNING:
```
INFO[0000] Using ECS task definition                     TaskDefinition="efe-test:1"
WARN[0001] Failed to create log group tutorial in us-east-1: The specified log group already exists 
INFO[0001] Created an ECS service                        service=efe-test taskDefinition="efe-test:1"
INFO[0002] Updated ECS service successfully              desiredCount=1 force-deployment=false service=efe-test
```

Removed.